yt_dlp/extractor/crunchyroll.py

```diff
@@ -53,13 +53,15 @@ def _set_auth_info(self, response):
         CrunchyrollBaseIE._AUTH_EXPIRY = time_seconds(seconds=traverse_obj(response, ('expires_in', {float_or_none}), default=300) - 10)
 
     def _request_token(self, headers, data, note='Requesting token', errnote='Failed to request token'):
-        try:  # TODO: Add impersonation support here
+        try:
             return self._download_json(
                 f'{self._BASE_URL}/auth/v1/token', None, note=note, errnote=errnote,
-                headers=headers, data=urlencode_postdata(data))
+                headers=headers, data=urlencode_postdata(data), impersonate='chrome')
         except ExtractorError as error:
             if not isinstance(error.cause, HTTPError) or error.cause.status != 403:
                 raise
+            if target := error.cause.response.extensions.get('impersonate'):
+                raise ExtractorError(f'Got HTTP Error 403 when using impersonate target "{str(target)}"')
             raise ExtractorError(
                 'Request blocked by Cloudflare; navigate to Crunchyroll in your browser, '
                 'then pass the fresh cookies (with --cookies-from-browser or --cookies) '
```

Comment:
do we want to mention impersonation in this message as well? or is the `_request_webpage` warning good enough?

Reply:
Copying the full message here isn't reasonable, but maybe a hint like `Install an impersonation target or pass fresh cookies (with --cookies-from-browser or --cookies) after navigating to Crunchyroll in your browser`